packages/storybook8/stories/Concepts/TeamsInterop/ComplianceBanner/ComplianceBanner.stories.tsx

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Stack } from '@fluentui/react';
+import { Meta } from '@storybook/react';
+import React from 'react';
+
+import { ComplianceBanner as Banner } from './snippets/ComplianceBanner.snippet';
+
+const ComplianceBannerStory = (args): JSX.Element => {
+  return (
+    <Stack style={{ minWidth: '50%', minHeight: '50%' }}>
+      <Banner callRecordState={args.callRecordState} callTranscribeState={args.callTranscribeState} />
+    </Stack>
+  );
+};
+
+export const ComplianceBanner = ComplianceBannerStory.bind({});
+
+export default {
+  title: 'Concepts/Teams Interop/Compliance Banner',
+  component: ComplianceBanner,
+  argTypes: {
+    callRecordState: { control: 'boolean', defaultValue: false, name: 'Is recording on?' },
+    callTranscribeState: { control: 'boolean', defaultValue: false, name: 'Is transcription on?' }
+  }
+} as Meta;
```

Comment:
need to set default values using `args: {}` 
<img width="798" alt="Screenshot 2024-07-29 at 3 42 18 PM" src="https://github.com/user-attachments/assets/1b02e535-f6cd-4335-a242-3dca918bd1f7">

Reply:
added it in now